tests/foreman/api/test_discoveredhost.py

```diff
@@ -228,6 +229,7 @@ def test_positive_provision_pxe_host(
     @pytest.mark.upgrade
     @pytest.mark.e2e
     @pytest.mark.on_premises_provisioning
+    @pytest.mark.parametrize('module_provisioning_sat', ['discovery'], indirect=True)
```

Comment:
this param seems useless in the below test for discovery, so instead of such parametrization, I think it would be nice if we could make a discovery pytest mark and use that condition as request.node.get_closest_marker(name='discovery') similar to destructive sat or use different test fixture for discovery, wdyt? (non-blocking atm)